rpc_jobs/rpc_artifact_build.yml

```diff
@@ -89,130 +89,130 @@
           cancel-builds-on-update: true
     dsl: |
       library "rpc-gating@${{RPC_GATING_BRANCH}}"
-      image_list = [
-        master: [
-            "Ubuntu 16.04.2 LTS prepared for RPC deployment"
-        ],
-        newton: [
-            "Ubuntu 16.04.2 LTS prepared for RPC deployment",
-            "Ubuntu 14.04.5 LTS prepared for RPC deployment"
-        ]
-      ]
-      env.trigger="{ztrigger}"
-      env.series="{series}"
-      common.shared_slave() {{
-        try {{
-          currentBuild.result = "SUCCESS"
-          // We need to checkout the rpc-openstack repo on the CIT Slave
-          // so that we can check whether the patch is a docs-only patch
-          // before allocating resources unnecessarily.
-          common.prepareRpcGit("auto", env.WORKSPACE)
-          if(common.is_doc_update_pr("${{env.WORKSPACE}}/rpc-openstack")){{
-            return
-          }}
+        common.globalWraps(){{
```

Comment:
Indentation here is off.

Reply:
Done

Reply:
Done

Reply:
DOUBLE DONE, just the way @major likes it.